Contributing/How-to-Contribute-Code.md

```diff
@@ -24,4 +21,4 @@
     * Ensure your Pull-Request passes all checks and has no conflicts
```

Comment:
Should we have a bit more info here? 
I want to keep things as simple as possible but on the other hand we need to be complete in order to avoid confusion.  So do you want to mention branches here or not?
Thx

Reply:
I agree that we should document our branching model. But I was not sure what to add since the maintainers have not weighed-in on this yet.